README.md

```diff
@@ -46,26 +48,338 @@ ember install ember-resources
+- [ember-array-map-resource](https://github.com/NullVoxPopuli/ember-array-map-resource) - provides a useArrayMap function which returns a resource that reactively maps data per-element, so that when the overall collection is dirtied, only the changed/new/removed elements affect the mapped collection
+- [ember-use-sound](https://github.com/chrismllr/ember-use-sound) - a resource for interacting with audio files
+
+## What is a Resource?
+
+> They are most similar to class-based Ember helpers, but with a more targeted goal overall. Resources are meant to bridge a gap between imperative programming and declarative programming.
+>
+> Ember templates are declarative. When we design a component, like the profile component from our previous example, we are specifying declaratively the HTML that should be rendered. If the data used in the templates ever updates, then Ember will update the rendered output as well, and we don't have to worry about the details. We don't have to tell Ember which specific steps to take, and when - it figures everything out for us.
+
+<div style="width: 100%; text-align: right;">
+  <cite>pzuraq on "<em>Introducing @use</em>"</cite>
+</div>
+
+So.. _what_ is a Resource, really?
+
+A _Resource_ is a behavior that can be used in both templates and javascript.
+
+### In JavaScript
+
+A Resource is an alternate API for
+```js
+import { cached } from '@glimmer/tracking';
+
+class A {
+  @cached
+  get myResource() {
+    return new MyResource(this.args.foo)
+  }
+}
+```
+In this example, `myResource` returns an instance of a class and will re-create that
+class if `@foo` changes. That class can have its own internal state.
+
+### In Templates
+
+A Resource is a stateful helper:
+```hbs
+{{#let (my-resource @foo) as |myResource|}}
+  {{!-- ... --}}
+{{/let}}
+```
+In this example, `my-resource` returns an instance of a class and will re-create that
+class if `@foo` changes. That class can have its own internal state and is available
+for use via the local variable `myResource`.
+
+### By example
+
+Throughout all of these examples, you may notice that some _can be_ a bit of work,
+maybe more suited towards libraries, rather than end-user app code.
+It's still good to know the problem space, because when your developing an app, you
+may not have the time to create a library so that your app's code is minimal.
+
+#### How do you deal with async data?
+
+Async data is a common problem in user interfaces.
+In Ember, it's standard practice to load _minimally required_
+data for in a route, and any secondary or supplementary data in components.
+
+For more information on thinking about async data,
+_[Async data and autotracking in ember octane](https://v5.chriskrycho.com/journal/async-data-and-autotracking-in-ember-octane/)
+by Chris Krycho_ may be of interest.
+
+For example, if you wanted to lazily and reactively load data based on
+an `@argument` passed to a component
+
+<details><summary>In vanilla Ember, no addons</summary>
```

Comment:
I like the collapsibles, but it's just hard to notice when scanning the rendered document. I wonder if having sub-sub-section header right above each of them would help? Just a thought, feel free to ignore.

Reply:
imo, there is too much in this README to _not_ have collapsibles for secondary information. Imagine in the following examples there were also 200+  lines of code

Reply:
I love the collapsibles. I just wanted pointed out that while scanning the document, they are easy to miss. Ultimately an issue with how browsers / github decided to style those.

Reply:
Yeah. I'm gonna try to figure out how to make this it's own page, so the examples aren't needed